views/modelIndex/grid.jade

```diff
@@ -0,0 +1,25 @@
+.body
+    div(class="table-responsive")
+        table.table.table-hover
+                thead
+                    tr
+                        th #
+                        if model.list.groupActions && model.list.groupActions.length
+                            th
+                                input(type="checkbox", data-linz-control="checked-all")
+                        if permissions.canDelete !== false || permissions.canEdit !== false || model.list.recordActions.length
+                            th.actions Actions
+                        for field in model.list.fields
+                            th= field.label
+                tbody
+                    for record, index in records
+                        tr
+                            td= from + index + 1
+                            if model.list.groupActions && model.list.groupActions.length
+                                td
+                                    input(type="checkbox", name="ids[]", value="#{record._id}", data-linz-control="checked-record")
+                            if permissions.canDelete !== false || permissions.canEdit !== false || model.list.recordActions.length
```

Comment:
@smebberson Actually now that I look at it, we can remove the check altogether.
We have moved the record-actions into a separate template so if any logic is needed, it can be managed there?